controllers/userController.js

```diff
@@ -68,5 +68,14 @@ const deleteUserData = async (req, res) => {
     res.status(500).send({ message: err })
   }
 }
+const getAllUsersData = async (req, res) => {
+  const { page } = req.params
+  try {
+    const userData = await userModel.findAll({ limit: 10, offset: (page - 1) * 10 })
```

Comment:
Why you use here limit 10 suppose If I want only 5 user of 1st page then what to do?